README.md

```diff
@@ -16,7 +16,9 @@ A list of resources connected to the [SpaceTraders API](https://spacetraders.io/
 ## Client
 * [Deliverance](https://github.com/Stumblinbear/Deliverance) - VueJS-based web interface for Space Traders.
 
-- [Trade Warriors](https://thaurin.github.io/trade-warriors/) - Trade Warriors is a frontend for the Space Traders API, written in Vue 3.
+- [TradeCommander](https://github.com/DotEfekts/TradeCommander/) - A browser-based CLI style interface written in Blazor WebAssembly.
```

Comment:
Since you changed the markdown list symbol to `*` for Trade Warriors let's try to be consistent in how we define lists. This issue is on me since I did not specify how to add the lists in the [contributing.md](https://github.com/SpaceTradersAPI/awesome-spacetraders/blob/main/contributing.md) file. I'll add information there being more precise how to add a new row in the future!

```suggestion
* [TradeCommander](https://github.com/DotEfekts/TradeCommander/) - A browser-based CLI style interface written in Blazor WebAssembly.
```